### DISCUSSION TITLE
Scaling Up Development on GitHub for a Custom Packaging Site: Tips and Pitfalls

### DISCUSSION THREAD

**Discussion by Dorothy-Reyes** [pybamm-team/PyBaMM#3348] [General]
Hi,
We're expanding our custom packaging site's development team and I'm interested in learning about strategies, tools and common challenges faced when scaling up on GitHub. What advice can the community provide to ensure a smooth transition to a larger development team?

**Comment by brosaplanella** [comment]
Please see https://github.com/pybamm-team/PyBaMM/discussions/3353#discussioncomment-7032538
